src/pages/index.js

```diff
@@ -15,9 +17,18 @@ export default function Home() {
         <meta name="description" content="Generated by create next app" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
+
+      </Head> 
+      
+      <main>
+        <CustomizableAside />
+      </main> 
+      
+
       </Head>
```

Comment:
@Sheyitofunmi you appear to have 2 closing `</Head> tags. 🤔 

Does your local app render correctly in the browser?

Reply:
Yes, I will figure out the tags and select one